services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/qrcode/QrCodeTemplateLoader.java

```diff
@@ -0,0 +1,32 @@
+package app.coronawarn.server.services.distribution.assembly.qrcode;
+
+import com.google.protobuf.ByteString;
+import java.io.IOException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+@Component
+public class QrCodeTemplateLoader {
+
+  private static final Logger logger = LoggerFactory.getLogger(QrCodeTemplateLoader.class);
+
+  public ByteString loadAndroidTemplateAsBytes() {
+    return loadPosterTemplate("pt-android-poster-1.0.0.xml");
+  }
+
+  public ByteString loadIosTemplateAsBytes() {
+    return loadPosterTemplate("pt-ios-poster-1.0.0.pdf");
```

Comment:
please create follow up tasks to load from VAULT environment - but for now it's OK